src/index.css

```diff
@@ -153,4 +153,9 @@ a:-webkit-any-link {
 .css-1u4zpwo-MuiSvgIcon-root-MuiStepIcon-root.Mui-completed {
 
   color: #fcdd06 !important;
+}
+
+.css-dcw4yf-MuiButtonBase-root-MuiButton-root:hover {
```

Comment:
We can not depend on this selector to apply our styles, I prefer if we created a styled component and add those changes to it